src/core/sdk-exceptions.ts

```diff
@@ -63,3 +63,10 @@ export function createExpectedBearerStringError() {
     'Expected argument to be a string in the `Bearer {token}` format.',
   );
 }
+
+export function mintingError() {
+  return new MagicAdminSDKError(
+    ErrorCode.MintingError,
+    'There was an error while minting. Check your contract ID and token ID (ERC1155 only)'
```

Comment:
Seems this also supports `ERC721`?
```suggestion
    'There was an error while minting. Check your contract ID and token ID (ERC721 and ERC1155 only)'
```

Reply:
Oh this is bad phrasing -- the tokenID is only present / required on ERC1155. I'll update to be more clear